### PR TITLE
[WFLY-15734] / [WFLY-15356] Remove security-plugins dependency from EJB3.

### PR DESCRIPTION
**Pull request by darranl**
https://issues.redhat.com/browse/WFLY-15734

This PR is just removing some code which should no longer be in use for legacy security integration so we can delete the legacy security modules and get the PicketBox dependency removed.

### CODE REVIEW

**Comment by darranl** [comment]
@tadamski / @chengfang Would it be possible to please get a review on this one?

File: ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java [hunk 1]
```diff
@@ -3250,4 +3250,8 @@
     @LogMessage(level = WARN)
     @Message(id = 529, value = "Failed to retrieve info from database for timer: %s")
     void failedToRetrieveTimerInfo(final TimerImpl timer, @Cause Exception e);
+
+    @Message(id = 530, value = "The deployment is configured to use legacy security which is no longer supported.")
+    IllegalStateException legacySecurityUnsupported();
+
```
Comment (chengfang) [inline ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:3256]: is it possible to include some context info like the security domain name used?

Comment (darranl) [reply ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java:3256]: Thanks @chengfang I will have a look if I can add this.

**Comment by darranl** [comment]
@chengfang I have added the name of the security domain to the log message, if you are Ok I think I can get this through tomorrow.

**Review by chengfang — APPROVED** [review]

**Pull request merged**